controller/app/index.js

```diff
@@ -77,21 +79,65 @@ function gotTable(data) {
         return Object.assign({ sched, task }, item)
     }).filter((item) => item.Status == 1 || item.sched.Schedule)
     if (firstLoad) {
+        let stringify = (d) => JSON.stringify(d, null, 2);
+        $("#taskTable").bootstrapTable({
+            idField: 'UUID',
+            columns: [
+                {title:'ID', field:'UUID'},
+                {title:'Task', field:'task', formatter: stringify},
+                {title:'Schedule', field:'sched', formatter: stringify},
+                {title:'Delete', field: 'operate', align: 'center', formatter: operate, events: { 'click .remove': cancel}}
+            ],
+            data: processedData,
+        });
+    } else {
+        $("#taskTable").bootstrapTable('load', processedData)
+    }
+}
+
+function gotRegions(data) {
+    if (firstLoad) {
+        let regions = data["regions"]
+        $.each(regions, (reg) => {
+            let li = $('<li/>').appendTo($("#regionlist"));
+            let button = $('<button/>')
+                .addClass("list-group-action")
+                .text(regions[reg])
+                .click((regionid) => {
+                    fetch(`./regions/${regions[reg]}`, { method: "GET", headers: getHeaders()}).then((response) => response.json()).then(gotDaemons)
+                })
+                .appendTo(li);
+        });
+    }
+}
+
+function gotDaemons(data) {
+    $("#botlist").find($("li")).remove()
+    let daemons = data["daemons"]
+    $.each(daemons, (daemon) => {
+        let li = $('<li/>').appendTo($("#botlist"));
+        let button = $('<button/>')
+            .addClass("list-group-action")
+            .text(daemons[daemon]["id"])
+            .click((daemonid) => {
+                fetch(`./regions/${daemons[daemon]["region"]}/${daemons[daemon]["id"]}`, { method: "GET", headers: getHeaders()}).then((response) => response.json()).then(showDaemon)
+            })
+            .appendTo(li);
+    });
+}
+
+function showDaemon(data) {
     let stringify = (d) => JSON.stringify(d, null, 2);
-    $("#taskTable").bootstrapTable({
-        idField: 'UUID',
+    $("#botdetail").bootstrapTable({
+        idField: 'id',
         columns: [
-            {title:'ID', field:'UUID'},
-            {title:'Task', field:'task', formatter: stringify},
-            {title:'Schedule', field:'sched', formatter: stringify},
-            {title:'Delete', field: 'operate', align: 'center', formatter: operate, events: { 'click .remove': cancel}}
+            {title:'ID', field:'id'},
+            {title:'Region', field:'region'},
+            // {title:'Wallet', field:'wallet', formatter: stringify},
+            {title:'Wallet', field:'wallet.address'},
         ],
-        data: processedData,
+        data: [data],
     });
-    firstLoad = false
```

Comment:
where is this getting set to false now?

Reply:
it's just up near the top of the file.

Reply:
nvm this is actually an issue we need to address.